tests/test_surface.py

```diff
@@ -0,0 +1,127 @@
+import os
+
+import oddt
+from oddt.surface import (generate_surface_marching_cubes,
+                          find_surface_residues)
+from sklearn.utils.testing import (assert_array_equal,
+                                   assert_equal)
+from nose.tools import assert_raises
+import numpy as np
+from skimage import __version__ as skimageversion
+
+test_data_dir = os.path.dirname(os.path.abspath(__file__))
+protein = next(oddt.toolkit.readfile('pdb', os.path.join(
+    test_data_dir, 'data/dude/xiap/receptor_rdkit.pdb')))
+protein.protein = True
+protein.addh(only_polar=True)
+
+def test_generate_surface_marching_cubes():
+    """Tests generating surfaces"""
+    verts1, faces1 = generate_surface_marching_cubes(protein, scaling=1., probe_radius=1.4, remove_hoh=False)
+    verts2, faces2 = generate_surface_marching_cubes(protein, scaling=2., probe_radius=1.4, remove_hoh=False)
+    verts3, faces3 = generate_surface_marching_cubes(protein, scaling=1., probe_radius=1.4, remove_hoh=True)
```

Comment:
I think the protein has no HOHs

Reply:
I think we should:
* check whether `remove_hoh=True` changes anything (it shouldn't)
* test this function on a structure with some water residues